hooks/pre-commit.js

```diff
@@ -1,6 +1,16 @@
 const {checkToken} = require("./common");
 const {exec} = require("./exec");
 
+const ErrorMessages = {
+  AVOID_DIRECT_IMPORT_FROM_PACKAGES:[
+    `Direct import from a package.`,
+    `This check catches direct imports such as:`,
+    `import { milliseconds } from "@dendronhq/common-all/lib/timing";`,
+    `While the above import should look as:`,
+    `import { milliseconds } from "@dendronhq/common-all";`
```

Comment:
nitpick: could be more generic so as to not be confused as an actual match that blocked the commit.

Reply:
Noted, made the message more generic in pull request v2